vergen-version/src/lib.rs

```diff
@@ -1,8 +1,11 @@
 /// Return git semver version
```

Comment:
I think we can rename this module to `version` or similar.

Reply:
yes, indeed! just wanted to submit as draft first to see whether this actually gives us anything!